frontend/src/components/Dropdown.js

```diff
@@ -10,47 +10,45 @@ import './Dropdown.css';
 
 const styles = {
   root: {
-  	paddingLeft: 8,
+    paddingLeft: 8,
   },
 };
 
 class Dropdown extends Component {
-	handleChange = event => {
-		this.props.changeValue(event.target.value);
-	}
-
-	renderMenuItems() {
```

Comment:
Note to @JamesZhaoLu 
The map function returns a new array so you could just write
```
return this.props.dropdownVals.map( val => {
  return <MenuItem value={ val }>{ val }</MenuItem>
})
```